### PR TITLE
First abs build

### PR DESCRIPTION
**Pull request by mcg1969**
- added description, dev_url, doc_url, license_family

### CODE REVIEW

**Comment by mcg1969** [comment]
ioption

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/dcleblanc/SafeInt/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 62fef99873ad975ddd8356923b3d51ed316209c1a05ac85814219373a13ae4d5
+  sha256: {{ sha256 }}
 
 build:
   number: 1
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: We should reset build number to 0.

Comment (mcg1969) [reply recipe/meta.yaml]: oops, of course

Comment (mcg1969) [reply recipe/meta.yaml]: @skupr-anaconda done

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**